frontend/src/App.css

```diff
@@ -0,0 +1,38 @@
+.App {
+  text-align: center;
+}
+
+.App-logo {
+  height: 40vmin;
+  pointer-events: none;
+}
+
+@media (prefers-reduced-motion: no-preference) {
+  .App-logo {
+    animation: App-logo-spin infinite 20s linear;
+  }
+}
```

Comment:
Quite a few unused CSS stuff here, lets keep it if the intent was to use them later or remove & add them when we get around to it.

Reply:
This is a byproduct of the create-react-app template app. It can be removed.